linux_os/guide/system/accounts/accounts-restrictions/password_storage/no_empty_passwords/bash/shared.sh

```diff
@@ -11,8 +11,21 @@ for FILE in ${NULLOK_FILES}; do
    sed --follow-symlinks -i 's/\<nullok\>//g' ${FILE}
 done
 {{% elif 'ubuntu' in product %}}
-FILE="/etc/pam.d/common-password"
-sed -i 's/\(.*pam_unix\.so.*\)\s\<nullok\>\(.*\)/\1\2/g' ${FILE}
+{{{ bash_pam_unix_enable() }}}
+config_file="/usr/share/pam-configs/unix"
```

Comment:
I think this should be `cac_unix`

Reply:
Solved in [4b4714b](https://github.com/ComplianceAsCode/content/pull/12660/commits/4b4714b4879fa7c799fd4091f96af8cb721c8acd)